commands/exportcommands.py

```
@@ -189,39 +222,65 @@ def _export_infrastructure_scripts(self, staging_dir):
+                '--exclude=claude-bot-commands/',
+                '--exclude=coding_prompts/',
+                '--exclude=prototype/',
+                '--exclude=tasks/',
+            ]
+
+            cmd = ['rsync', '-av'] + exclude_patterns + [f"{source_dir}/", f"{target_dir}/"]
+
+            result = subprocess.run(cmd, capture_output=True, text=True)
+            if result.returncode != 0:
+                raise Exception("rsync failed, using fallback")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)